### DISCUSSION TITLE
How is the calc size calculated

### DISCUSSION THREAD

**Discussion by barrisam** [ChartsCSS/charts.css#76] [Q&A] [answered]
![calc 2](https://user-images.githubusercontent.com/40701426/226097370-ddb178c3-b128-409a-a25a-34fc6b2b4e4b.PNG)
![calc](https://user-images.githubusercontent.com/40701426/226097372-0971c357-698b-4624-afab-fffb1994d360.PNG)

I attached images underlining the section i am confused about. I am not sure how the style: size calc is calculated for each bar size of a chart. I would like more explanation on that

**Comment by ramiy** [comment, marked as answer]
The `--size` have to be a number between `0` to `1`. You don't have to use the `calc()` function, simply use `--size: 0.196;`.
